### ISSUE TITLE
[REQUEST] Scheduled restart monitor's IP dynamic

### ISSUE THREAD

**Issue by MikeTheCleric** [Odianosen25/Monitor-App#43]
Hi, when i configured your monitor app I've noticed that in the section called "scheduled_restart" the IP of every RPIzero is static. My nodes use DHCP and every time I reboot them I must modify the config. There's a way to use a symbolic name instead of the IP?? Or read IP addresses after the reboot and write them in the config??

Thanks in advice for any answer.

**Comment by Odianosen25** [comment]
Hello @MikeTheCleric,

Unfortunately I have no solution to that yet I am afraid,  as it is on my todo list to support the use of host name or MAC address of the Zeros themselves.

Till I implement that, or someone helps with a PR, there is no solution to that unfortunately.

Regards

**Comment by skynet01** [comment]
@MikeTheCleric you can just add the mac address of your RPIzero to your router so it always gets the same IP address.

**Comment by Odianosen25** [comment]
@skynet01 yeah, but the reason I actually wanted to add it, was for those that have no access to the router. But thinking of it, if one can setup monitor, I am damn sure they can access the router. On the other hand, the reason for this app was to make things simple, so will still try to add it when I figure out how to do it.

Regards

**Comment by skynet01** [comment]
Very true, good point :)
